internals/secrethub/acl_check_test.go

```diff
@@ -108,12 +108,15 @@ func TestACLCheckCommand_Run(t *testing.T) {
 			io := ui.NewFakeIO()
 			tc.cmd.io = io
 
-			lister := &tc.lister
+			lister := tc.lister
 
 			tc.cmd.newClient = func() (secrethub.ClientInterface, error) {
 				return fakeclient.Client{
 					AccessRuleService: &fakeclient.AccessRuleService{
-						LevelLister: lister,
+						ListLevelsFunc: func(path string) ([]*api.AccessLevel, error) {
+							assert.Equal(t, path, tc.listerArgPath)
```

Comment:
What happens if `List` is never called? Then this assertion is never made.

I believe we need to either use a struct to store the value or declare a variable in the scope where the assertion was previously made and check there. What do you think?

Reply:
Yes, I accidentally missed that :sweat_smile:
I agree, that seems like a good approach. Fixed it!